gst/mqtt/ntputil.c

```diff
@@ -119,6 +119,15 @@ const int64_t NTPUTIL_SEC_TO_USEC_MULTIPLIER = 1000000;
 const char NTPUTIL_DEFAULT_HNAME[] = "pool.ntp.org";
 const uint16_t NTPUTIL_DEFAULT_PORT = 123;
 
+/**
+ * @brief Wrapper function of ntohl.
+ */
+uint32_t
+_convert_to_host_byte_order (uint32_t in)
+{
+  return ntohl (in);
```

Comment:
Question.
I can't find `#include <netinet/in.h>` or `#include <arpa/inet.h>` in this file.
Should we include one of them instead of `#include <netinet/ip.h>`?

Reply:
I changed to include <arpa/inet.h> instead of <netinet/ip.h>. Thanks.